app/webpacker/components/Tickets/TicketWorkbenches/EditPersonTicketWorkbench.jsx

```diff
@@ -49,6 +79,9 @@ function EditPersonTicketWorkbenchForWrt({ ticketDetails, actingStakeholderId, s
       <EditPersonValidations
         ticketDetails={ticketDetails}
       />
+      <EditPersonRequestedChangesList
+        requestedChanges={ticketDetails.ticket.metadata?.tickets_edit_person_fields}
```

Comment:
I assume `ticket` is already available, so `ticket?.metadata?...` should also work I guess.